api/src/routing.js

```diff
@@ -250,6 +252,7 @@ app.all('/+admin(/*)?', authorization.handleAuthErrors, authorization.offlineUse
 
 app.use(express.static(environment.staticPath));
 app.use(express.static(environment.webappPath));
+app.get('/extension-libs/:name', extensionLibs.get);
```

Comment:
Yeah I'm not 100% happy with this name, but naming is hard. I feel `extensions` is too generic, and as we get into more extensibility may complicate things. This is exclusively for separate JS files, so libs felt write, but maybe `extension-scripts`? Maybe it's the `extension` bit that's the problem and it should be `configured-scripts`? I haven't found anything I really like yet...

Reply:
> This is exclusively for separate JS files, so libs felt write

Ahh, I am understanding more of your reasoning behind that name!  This is pure opinion, but to me `extension-scripts > extension-libs > configured-scripts` but honestly it's a fairly weak preference and I am not sure it would be worth going back and re-naming everything.  I am fine with `extension-libs` if we cannot come up with anything else that is clearly better!